prow/cmd/phaino/local.go

```diff
@@ -241,10 +241,12 @@ func convertToLocal(log *logrus.Entry, pj prowapi.ProwJob, allowPrivilege bool)
 		}
 	}
 	if workingDir == "" {
+		workingDir = container.WorkingDir
+	}
+	if workingDir != "" {
 		workingDir = "/random-location"
```

Comment:
If `workingDir` is specified we don't want to overwrite it.

Reply:
Done